.github/workflows/build-build-tools-image.yml

```diff
@@ -78,14 +79,17 @@ jobs:
           provenance: false
           push: true
           pull: true
-          file: Dockerfile.build-tools
-          cache-from: type=registry,ref=cache.neon.build/build-tools:cache-${{ matrix.arch }}
-          cache-to: ${{ github.ref_name == 'main' && format('type=registry,ref=cache.neon.build/build-tools:cache-{0},mode=max', matrix.arch) || '' }}
-          tags: neondatabase/build-tools:${{ inputs.image-tag }}-${{ matrix.arch }}
+          file: docker/debian/${{ matrix.os }}/Dockerfile.build-tools
```

Comment:
Instead of copying files, can we pass the base image (`debian:bookworm-slim`/`debian:bullseye-slim`) as an argument?

Reply:
We have https://github.com/neondatabase/neon/blob/ac6a1151aeaf8fcc007b833a97dca57328c44d05/Dockerfile.build-tools#L9
So by setting it to the correct version it might work